habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -441,6 +490,10 @@ def get_robot_spawns(
         if target_distance > distance_threshold or not is_navigable:
             continue
 
+        island_radius = sim.pathfinder.island_radius(start_position)
```

Comment:
Can you add a comment of what this does?

Reply:
Same here! Thank you, Andrew!

Reply:
Same here! Thank you, Andrew!

Reply:
Removed this.